readme.md

```diff
@@ -9,27 +9,6 @@ With NodeJS installed, just run NPM:
 
 > npm install can-compile -g
 
-## Command line
```

Comment:
I guess I would like people to read about the gulp/grunt tasks before the command line. Just a difference of preference. I can move it back up if you like.